app/src/main/java/com/example/myapplication/network/data/UserAuth.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.myapplication.network.data
+
+import com.google.gson.annotations.SerializedName
+
+data class UserAuth(
+    @SerializedName("email") val email: String,
```

Comment:
If the json key value is the same as your variable's name, there's no need to use SerializedName. Retrofit is smart enough to map it.